### PR TITLE
add makeReadOnly() to EXPLAINER

### PR DESCRIPTION
**Pull request by zolkis**
Signed-off-by: Zoltan Kis <zoltan.kis@intel.com>

### CODE REVIEW

File: EXPLAINER.md [hunk 1]
```diff
@@ -53,6 +53,10 @@ post-it notes, anyone can read them, and unless read-only, anyone can write
 them. Web NFC does not handle automatic encryption or signing of NFC content
 (with NDEF signatures): handling PKI infrastructure is left to applications.
 
+The API does not support controlling who can change the content on an NDEF
+tag. However, the API supports making an NDEF tag permanently read-only, see the
+[examples](https://w3c.github.io/web-nfc/#make-an-nfc-tag-permanently-read-only).
+
 ## API shape and examples
```
Comment (beaufortfrancois) [inline EXPLAINER.md:82]: Shall we add a simple JS example in this section?

```js
const ndef = new NDEFReader();
try {
  await ndef.write("Hello world");
  console.log("Message written.");
  await ndef.makeReadOnly();
  console.log("NFC tag has been made permanently read-only after writing to it.");
} catch (error) {
  console.log(`Operation failed: ${error}`);
}
```

File: EXPLAINER.md [hunk 2]
```diff
@@ -284,6 +290,13 @@ Since the merge of `NDEFWriter` and `NDEFReader`, the former `onerror` event on
 relevant on writes, which is not the case. This change clarifies the error is
 only relevant for scans.
 
+### Added the `makeReadOnly()` method
+
+Justified in [issue 558](https://github.com/w3c/web-nfc/issues/558) and by feedback
+received on Twitter. It seems like web developers would like to be able to
```
Comment (beaufortfrancois) [inline EXPLAINER.md]: Can we add one or two examples use cases there as well?

File: EXPLAINER.md (hunk 1 above)
Comment (zolkis) [reply EXPLAINER.md:82]: I thought that referring to the examples in the spec was fine, but we can have it inline, too, indeed.

Comment (zolkis) [reply EXPLAINER.md:82]: Actually in other sections we also only link the examples.
Inline examples are there only for the integrations.
I think it makes sense this way (linking examples).

Comment (kenchris) [reply EXPLAINER.md:82]: I think you should be able to read an explainer without having to open or link into other documents

File: EXPLAINER.md (hunk 2 above)
Comment (kenchris) [reply EXPLAINER.md]: I don't like wording like "seems like" - so we don't really know? It's very weak

File: EXPLAINER.md (hunk 1 above)
Comment (zolkis) [reply EXPLAINER.md:82]: Should we inline the other examples as well, then?

Comment (kenchris) [reply EXPLAINER.md:82]: I would say yes

**Comment by zolkis** [comment]
The previous commit amended, with example and use case for readonly. Feel free to further change it.

**Review by beaufortfrancois — APPROVED** [review]

**Pull request merged**